frontend/amundsen_application/static/js/interfaces/Resources.ts

```diff
@@ -55,6 +45,21 @@ export interface TableResource extends Resource {
   badges?: Badge[];
 }
 
+export interface FeatureResource extends Resource {
+  type: ResourceType.feature;
+  key: string;
+  name: string;
+  version: string;
+  feature_group: string;
+  entity: string;
+  data_type?: string;
+  description: string;
+  availability: string[];
+  last_updated_timestamp?: number;
+  created_timestamp?: number;
+  badges?: Badge[];
+}
```

Comment:
The FeatureResource is the data type returned by search results. Are all of these fields part of the search result?

Reply:
I am still waiting on Dmitriy to get search done so unclear so far